marimo/_utils/xdg.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2024 Marimo. All rights reserved.
+from __future__ import annotations
+
+import os
+from pathlib import Path
+
+
+def home_path() -> Path:
+    """Get home directory or temp directory if home directory is not available.
+
+    Returns:
+        Path: The home directory.
+    """
+    try:
+        return Path.home().resolve()
+    except RuntimeError:
+        # Can't get home directory, so use temp directory
+        return Path("/tmp")
```

Comment:
this is the only logic change (at least intentionally). instead of returning `None` in some code paths and bubbling up, we instead return a `tmp` dir

Reply:
i think this is a good place for this function